src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3194,7 +3194,7 @@ def create_vmss(cmd, vmss_name, resource_group_name, image=None,
                 user_data=None, network_api_version=None, enable_spot_restore=None, spot_restore_timeout=None,
                 capacity_reservation_group=None, enable_auto_update=None, patch_mode=None, enable_agent=None,
                 security_type=None, enable_secure_boot=None, enable_vtpm=None, automatic_repairs_action=None,
-                v_cpus_available=None, v_cpus_per_core=None, accept_term=None, disable_integrity_monitoring=False,
+                v_cpus_available=None, v_cpus_per_core=None, accept_term=None, enable_integrity_monitoring=False,
```

Comment:
Same as above

Reply:
@Bab95 --- please take a look at this.

Reply:
@Howie425  I have reviewed the PR. looks good to me.
instead of disable_integrity_monitoring enable_integrity_monitoring should be used 
i.e. Customer should choose to install the extension.

Reply:
Yes, actually, what I mean is that we should use `enable_integrity_monitoring` instead of `disable_integrity_monitoring`, but the definition of `disable_integrity_monitoring` in the custom method still needs to be preserved. Otherwise, if it is deleted now, it will directly cause breaking changes to the customer's automation scripts without any prior notification message.

Reply:
Done